mobile_verifier/src/service_provider/reward.rs

```diff
@@ -57,15 +53,15 @@ impl ServiceProviderRewardInfos {
         dc_sessions: ServiceProviderDCSessions,
         promotions: ServiceProviderPromotions,
         total_sp_allocation: Decimal, // Bones
-        mobile_bone_price: Decimal,   // Price in Bones
-        reward_epoch: Range<DateTime<Utc>>,
+        hnt_bone_price: Decimal,      // Price in Bones
+        reward_info: ResolvedSubDaoEpochRewardInfo,
```

Comment:
This is an example of somewhere that only uses `epoch_period`.

Reply:
addressed here: e8643f61bafa27c8c1a9680da8f625f068f01393